resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/BasicDriverFeatureStep.scala

```diff
@@ -170,4 +170,3 @@ private[spark] class BasicDriverFeatureStep(conf: KubernetesDriverConf)
     additionalProps.toMap
   }
 }
-
```

Comment:
I understand the intention, but let's not touch irrelevant part.

Reply:
ok